src/msw/mediactrl_am.cpp

```diff
@@ -1127,7 +1127,7 @@ bool wxAMMediaBackend::Load(const wxURI& location, const wxURI& proxy)
     if(pPlay)
     {
         pPlay->put_UseHTTPProxy(VARIANT_TRUE);
-        pPlay->put_HTTPProxyHost(wxBasicString(proxy.GetServer()).Get());
+        pPlay->put_HTTPProxyHost(wxBasicString(proxy.GetServer()).Detach());
```

Comment:
I think that all calls like this may actually leak a `BSTR`. I have no experience with COM but [the documentation](https://msdn.microsoft.com/en-us/library/xda6xzx7.aspx) reads (emphasis mine):

> When you call into a function that expects a BSTR argument, you must allocate the memory for the BSTR before the call and **release it afterwards**. 

Am I wrong?